tests/tasks/test_blended_skill_talk.py

```diff
@@ -402,5 +402,31 @@ def test_safe_personas(self):
             self.assertEqual(len(personas), count)
 
 
+class TestBlendedSkillTalkModels(unittest.TestCase):
+    def test_accuracies(self):
+        """
+        Test accuracies of BST models in the zoo.
+        """
+        model_names_to_desired_accuracies = {
+            'bst_single_task': 0.8125,
+            'convai2_single_task': 0.7715,
+            'ed_single_task': 0.7148,
+            'wizard_single_task': 0.6797,
+            'multi_task': 0.8086,
+            'multi_task_bst_tuned': 0.8457,
+        }
+        for model_name, desired_accuracy in model_names_to_desired_accuracies.items():
+            valid, _ = testing_utils.eval_model(
+                opt={
+                    'batchsize': 64,
+                    'model_file': f'zoo:blended_skill_talk/{model_name}/model',
+                    'task': 'blended_skill_talk',
+                    'num_examples': 1000,
```

Comment:
i would even suggest dropping this down to like 100 examples (cut from 9 minutes to under a min)

Reply:
Also why not a multiple of 64 ;)

Reply:
Also heads up, the GPUs in CircleCI are like 5 generations behind our devfairs. If it's 9 minutes on your devfair, it's going to be like 30 on the CI machines.

Reply:
Yes - reducing this to 64